app.ts

```diff
@@ -311,21 +307,24 @@ export class App<
   /**
    * Creates HTTP server and dispatches middleware
    * @param port server listening port
-   * @param Server callback after server starts listening
    * @param host server listening host
+   * @param Server callback after server starts listening
    */
-  async listen(port: number, cb?: () => void, hostname?: string) {
+  async listen(port: number, hostname?: string, cb?: (error: any) => void) {
```

Comment:
why did you change function signature? It should be the same as in node version:

```ts
listen(port?: number, cb?: () => void, host?: string): Server
```

Reply:
Normally, cbs tend to be the last argument. Such as `emitter.on` in node

Reply:
fixed it nonetheless.